app/controllers/v6/assessments_controller.rb

```diff
@@ -141,6 +145,28 @@ def parse_employment_details(employments)
       end
     end
 
+    def parse_employment_income(employments, submission_date)
+      employments.map do |employment|
+        employment_payments = employment[:payments].map do |payment|
+          EmploymentPayment.new(
+            date: Date.parse(payment[:date]),
+            gross_income: payment[:gross],
+            benefits_in_kind: payment[:benefits_in_kind],
+            tax: payment[:tax],
+            national_insurance: payment[:national_insurance],
+            client_id: payment[:client_id],
+          )
+        end
+        Employment.new(
+          name: employment[:name],
+          client_id: employment[:client_id],
+          receiving_only_statutory_sick_or_maternity_pay: employment[:receiving_only_statutory_sick_or_maternity_pay],
+          employment_payments:,
+          submission_date:,
```

Comment:
see other comments - don't think Employment should have a submission_date property